src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -129,18 +129,30 @@ static function (ChildDefinition $definition, AsTwigComponent $attribute) {
         ;
 
         $container->register('ux.twig_component.twig.lexer', ComponentLexer::class);
+        $container->getDefinition('ux.twig_component.twig.lexer')
+            ->addMethodCall('enableShortSyntax', [$config['short_syntax']]);
```

Comment:
to me:
- either call the method `enableShortSyntax` but with no parameter, and have the `addMethodCall` wrapped by an `if` using the config value
- either call the method `setShortSyntax` and keep the parameter

I'd favor the first personally